qiskit/visualization/text.py

```diff
@@ -459,7 +459,9 @@ def __str__(self):
     def _repr_html_(self):
         return '<pre style="word-wrap: normal;' \
                'white-space: pre;' \
-               'line-height: 15px;">%s</pre>' % self.single_string()
+               'line-height: 15px;' \
+               'font-family: SFMono-Regular,Consolas,Liberation Mono,Menlo,Courier,monospace">' \
```

Comment:
does this require having these fonts installed?

Reply:
No. They are options.